src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-lts-2021-upgrade-notes.md

```diff
@@ -120,6 +120,10 @@ Spreadsheet addon is now loaded by default but the button contribution is disabl
 ![Spreadsheet package enable button in Designer](nx_asset://b8fd28dd-0272-43b1-a083-dfde295c312b ?w=650,border=true)
 {{! /multiexcerpt}}
 
+### Rich Text Editor Uses QuillJS by Default
+
+<a href="https://quilljs.com" target="_blank">QuillJS</a> is used as the solution for the rich text editor in Web UI. This change is [available as an option](https://jira.nuxeo.com/browse/WEBUI-291) in LTS 2019 if you want your users to get accustomed to it before migrating.
```

Comment:
It is only an option for LTS2019. For LTS2021, quill is not optional.

Reply:
That's what I meant, probably didn't make it clear enough. Will rephrase, thanks!